src/helpers/updateHistoryRecord.js

```diff
@@ -0,0 +1,13 @@
+const updateHistoryRecord = (url = null, customData = {}) => {
+	url = url || window.location.href; // window.location.href.split(window.location.hostname)[1];
```

Comment:
Same as with `createHistoryRecord`. Or what was the reason the full URL is being used here?

Reply:
Done :) It was just inconsistent since it doesn't matter in this context if the URL is relative or absolute.